docs/changelog/108161.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 108161
+summary: Tests pass
```

Comment:
hmm, how about instead of the label `>bug` which generates this, we do `>non-issue`, `>tech-debt`? I haven't used tech debt before but we probably don't need a change log for this.

Reply:
seems like we still got a changelog, but I updated it to have a more accurate summary

Reply:
using `>tech-debt` seems to have failed the build